proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,63 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime to SDL Core for get real system time during security TLS handshake.
+
+
+## Motivation

```

Comment:
Suggested Text: 
Head units can make use of TLS handshake mechanism for security reasons to ensure the connected applications are authentic. This becomes necessary for projection experiences (such as Mobile Navigation) where the app has complete control over the content it displays. To facilitate such security mechanisms SDL Core will need to be provided with accurate time information. Today, SDL Core can get the time information by querying the platform OS which may or maynot be accurate. 
This proposal provides a solution to this problem by having SDL Core request the system for time information. The system would then be responsible for providing accurate time related information.

Reply:
@robinmk Fixed in https://github.com/smartdevicelink/sdl_evolution/pull/324/commits/19088cdb433d915b8fe826acfcf622b427a02f1e